todo-app/src/App.js

```diff
@@ -1,13 +1,40 @@
-import React, { Component } from 'react';
+// import React, { Component } from 'react';
 
-class App extends Component {
+// class App extends Component {
+//   render() {
+//     return (
+//       <div>
+//         Hello World!
+//       </div>
+//     );
+//   }
+// }
+
+// export default App;
+
+import React from 'react';
+import Tasks from './Task'
+import task from './taskDB'
+
+export default class App extends React.Component {
   render() {
     return (
       <div>
-        Hello World!
+        <h1>ToDo:</h1>
+        <div>
+          {task.map((task, index) => {
+            return (
+              <Tasks
```

Comment:
If this renders a single task then we should probably name it `Task` right? Also it would help a lot to keep naming consistent, for example instead of `Task` we name it `Todo` or `TodoItem`?

Reply:
Thanks for feedback . I need to think first before name it later .